CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
+## 0.8.14
+
+- Filter out `only` postfix tag for getSwaggerFiles function
```

Comment:
why is this?

Reply:
Some only tag doesn't contains all swagger files. That only suffix tag is  for some special purpose.